src/System.Security.Cryptography.X509Certificates/src/Resources/Strings.resx

```diff
@@ -115,9 +115,21 @@
   <data name="Cryptography_CertReq_DuplicateExtension" xml:space="preserve">
     <value>An X509Extension with OID '{0}' has already been specified.</value>
   </data>
+  <data name="Cryptography_CertReq_IssuerBasicConstraintsInvalid" xml:space="preserve">
+    <value>The issuer certificate does not have an appropriate value for the Basic Constraints extension.</value>
+  </data>
+  <data name="Cryptography_CertReq_IssuerKeyUsageInvalid" xml:space="preserve">
+    <value>The issuer certificate's Key Usage extension is present but does not contain the KeyCertSign flag.</value>
+  </data>
   <data name="Cryptography_CertReq_IssuerRequiresPrivateKey" xml:space="preserve">
     <value>The provided issuer certificate does not have an associated private key.</value>
   </data>
+  <data name="Cryptography_CertReq_NotAfterNotNested" xml:space="preserve">
+    <value>The requested notAfter value '{0}' is later than issuerCertificate.NotAfter ({1}).</value>
```

Comment:
You have the first date in quotes and the second in parens in both of these strings